superset-frontend/src/dashboard/components/menu/DownloadMenuItems/DownloadScreenshot.test.tsx

```diff
@@ -0,0 +1,198 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { render, screen, waitFor } from 'spec/helpers/testing-library';
+import userEvent from '@testing-library/user-event';
+import { Menu } from 'src/components/Menu';
+import fetchMock from 'fetch-mock';
+import { logging } from '@superset-ui/core';
+import { DownloadScreenshotFormat } from './types';
+import DownloadScreenshot from './DownloadScreenshot';
+
+const mockAddDangerToast = jest.fn();
+const mockLogEvent = jest.fn();
+const mockAddSuccessToast = jest.fn();
+const mockAddInfoToast = jest.fn();
+
+jest.spyOn(logging, 'error').mockImplementation(() => {});
```

Comment:
Why do we have code updates in this PR? Are we missing some cherries?

Reply:
should be fixed now